src/test/java/teammates/test/pageobjects/InstructorCommentsPage.java

```diff
@@ -31,7 +31,7 @@ public void loadResponseComments() throws Exception{
         browser.driver.findElement(By.xpath("//*[@id=\"panel_display-2\"]/div/div[1]")).click();
         waitForPageToLoad();
         try{
-            waitForElementPresence(By.xpath("//*[@id=\"panel_display-2\"]/div/div[2]/div[1]/div"));
+            waitForElementVisibility(browser.driver.findElement(By.xpath("//*[@id=\"panel_display-2\"]/div/div[2]/div[1]/div")));
```

Comment:
It's not really related to your pr, but is it possible to improve on the selector, or introduce a variable or comment to explain it? It's hard to figure out what <code>[@id=\"panel_display-2\"]/div/div[2]/div[1]/div"</code> is. We can do this in a new issue.